dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/validation/XtxtUMLModifierValidator.xtend

```diff
@@ -0,0 +1,38 @@
+package hu.elte.txtuml.xtxtuml.validation
+
+import hu.elte.txtuml.xtxtuml.xtxtUML.TUConstructor
+import org.eclipse.xtext.validation.Check
+
+import static hu.elte.txtuml.xtxtuml.validation.XtxtUMLIssueCodes.*
+import static hu.elte.txtuml.xtxtuml.xtxtUML.XtxtUMLPackage.Literals.*
+import hu.elte.txtuml.xtxtuml.xtxtUML.TUAttribute
+import hu.elte.txtuml.xtxtuml.xtxtUML.TUExternality
+
+class XtxtUMLModifierValidator extends XtxtUMLConnectorValidator {
```

Comment:
As constructors and attributes are named elements, it's advisable to include their names in the error messages, because in this way it's easier for the user to manage issues in the problems view. For an example, see _[XtxtUMLClassValidator.xtend:79](https://github.com/ELTE-Soft/txtUML/blob/5ea692b4cb20a19029876b3edccc1db6d916d78e/dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/validation/XtxtUMLClassValidator.xtend#L79-L80)_.